include/oneapi/dpl/experimental/kt/internal/gpu_radix_sort_submitters.h

```diff
@@ -30,16 +33,137 @@ template <typename _KeyT, ::std::uint8_t __radix_bits, ::std::uint32_t __stage_c
           typename _KernelName>
 struct __radix_sort_onesweep_histogram_submitter;
 
+template <typename _KeyT, ::std::uint32_t __radix_bits, ::std::uint32_t __stage_count,
+          ::std::uint32_t __hist_work_group_count, ::std::uint32_t __hist_work_group_size, bool __is_ascending,
+          typename... _Name>
+struct __radix_sort_onesweep_histogram_submitter<
+    _KeyT, __radix_bits, __stage_count, __hist_work_group_count, __hist_work_group_size, __is_ascending,
+    oneapi::dpl::__par_backend_hetero::__internal::__optional_kernel_name<_Name...>>
+{
+    template <typename _KeysRng, typename _GlobalOffsetData>
+    sycl::event
+    operator()(sycl::queue& __q, _KeysRng&& __keys_rng, const _GlobalOffsetData& __global_offset_data,
+               ::std::size_t __n, const sycl::event& __e) const
+    {
+        sycl::nd_range<1> __nd_range(__hist_work_group_count * __hist_work_group_size, __hist_work_group_size);
+        return __q.submit([&](sycl::handler& __cgh) {
+            oneapi::dpl::__ranges::__require_access(__cgh, __keys_rng);
+            __cgh.depends_on(__e);
+            auto __data = __keys_rng.data();
+            __cgh.parallel_for<_Name...>(__nd_range, [=](sycl::nd_item<1> __nd_item) {
+                __global_histogram<_KeyT, decltype(__data), __radix_bits, __stage_count, __hist_work_group_count,
+                                   __hist_work_group_size, __is_ascending>(__nd_item, __n, __data,
+                                                                           __global_offset_data);
+            });
+        });
+    }
+};
+
 template <::std::uint32_t __stage_count, ::std::uint32_t __bin_count, typename _KernelName>
 struct __radix_sort_onesweep_scan_submitter;
 
+template <::std::uint32_t __stage_count, ::std::uint32_t __bin_count, typename... _Name>
+struct __radix_sort_onesweep_scan_submitter<
+    __stage_count, __bin_count, oneapi::dpl::__par_backend_hetero::__internal::__optional_kernel_name<_Name...>>
+{
+    template <typename _GlobalOffsetData>
+    sycl::event
+    operator()(sycl::queue& __q, _GlobalOffsetData& __global_offset_data, ::std::size_t __n,
+               const sycl::event& __e) const
+    {
+        sycl::nd_range<1> __nd_range(__stage_count * __bin_count, __bin_count);
+        return __q.submit([&](sycl::handler& __cgh) {
+            __cgh.depends_on(__e);
+            __cgh.parallel_for<_Name...>(__nd_range, [=](sycl::nd_item<1> __nd_item) {
+                ::std::uint32_t __offset = __nd_item.get_global_id(0);
+                const auto __g = __nd_item.get_group();
+                ::std::uint32_t __count = __global_offset_data[__offset];
+                ::std::uint32_t __presum = sycl::exclusive_scan_over_group(__g, __count, sycl::plus<::std::uint32_t>());
```

Comment:
Could you replace it with:
`__dpl_sycl::__exclusive_scan_over_group(__g, __count, __dpl_sycl::__plus<::std::uint32_t>());`?

wrappers from `__dpl_sycl` namespace are used to provide backward compatibility with older versions of DPC++ compiler.

They are available in `sycl_defs.h` header.